netbox/dcim/models/devices.py

```diff
@@ -1522,3 +1522,10 @@ class Meta:
 
     def __str__(self):
         return str(self.mac_address)
+
+    @property
```

Comment:
Any reason not to use `@cached_property` here?

Reply:
I don't see why not.  Lemme make that change.